markdown_lab_rs.py

```
@@ -41,19 +53,175 @@
+        # Handle blockquotes
+        elif line.startswith(">") and not in_code_block:
+            document["blockquotes"].append(line[1:].strip())
+            
+        # Handle paragraphs (very simplified)
+        elif line.strip() and not in_code_block:
+            document["paragraphs"].append(line.strip())
+    
+    return document
+
+
+def document_to_xml(document: Dict) -> str:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)